core/src/main/java/org/apache/iceberg/actions/BaseRewriteDataFilesAction.java

```diff
@@ -252,7 +254,6 @@ public RewriteDataFilesActionResult execute() {
     return new RewriteDataFilesActionResult(currentDataFiles, addedDataFiles);
   }
 
-
```

Comment:
Please remove unnecessary changes like this and reformatting lines that don't need to change. We avoid changes that aren't required to avoid git commit conflicts.

Reply:
> Yeah this is definitely the right thing to do, we are handling this in the new code (
> 
> https://github.com/apache/iceberg/blob/a1bd63d56751999ecee89b871992d7bac395fd52/core/src/main/java/org/apache/iceberg/actions/RewriteDataFilesCommitManager.java#L97-L109
> 
> ) as well and while this path is deprecated now I think we should definitely fix it to avoid causing issues while the path is still available.

looking forward to using the new version rewrite